dietpi/dietpi-software

```diff
@@ -11615,6 +11655,62 @@ _EOF_
 			G_EXEC chown -R ffsync:ffsync /mnt/dietpi_userdata/firefox-sync
 
 		fi
+		
+                software_id=183 # Bitwarden RS
+                if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+                    Banner_Configuration
+
+                    Create_User -d /mnt/dietpi_userdata/bitwarden_rs bitwarden_rs
+
+                    # Make dir and data dir needed to run
+                    G_EXEC mkdir -p /mnt/dietpi_userdata/bitwarden_rs/data
+
+                    # Make and edit the template config to have no web vault
+                    G_EXEC cp /opt/bitwarden_rs/.env.template /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
```

Comment:
It's better to not overwrite existing environment on reinstall, isn't it? What about:
```
if [[ ! -f '/mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env' ]]
then
	G_EXEC cp /opt/bitwarden_rs/.env.template /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
	G_CONFIG_INJECT 'WEB_VAULT_ENABLED=' 'WEB_VAULT_ENABLED=false' /mnt/dietpi_userdata/bitwarden_rs/bitwarden_rs.env
fi
```

Reply:
Yep, that is sensible. I forgot about the reinstall command.

Reply:
The line above then needs to be removed 😄.

Reply:
Oops, fixed